.azure/obtemplates/build-winuser-xdp.yml

```diff
@@ -1,36 +0,0 @@
-parameters:
-  config: ''
-  tls: 'schannel'
-  platform: 'windows'
-
-jobs:
-- job: build_${{ parameters.platform }}_${{ parameters.tls }}_${{ parameters.config }}_xdp
-  displayName: ${{ parameters.platform }} ${{ parameters.tls }} ${{ parameters.config }} XDP
-  pool:
-    type: windows
-  variables:
-    ob_outputDirectory: $(Build.SourcesDirectory)\artifacts\bin\${{ parameters.platform }}
-    ob_sdl_binskim_break: true # https://aka.ms/obpipelines/sdl
-    ob_sdl_codeSignValidation_excludes: -|**\*.exe # Disable signing requirements for test executables
-  steps:
-  - task: PowerShell@2
-    displayName: Prepare Build Machine
-    target: windows_build_container2
-    inputs:
-      pwsh: true
-      filePath: scripts/prepare-machine.ps1
-      arguments: -InstallXdpSdk -ForContainerBuild
```

Comment:
Don't we need to update `prepare-machine.ps1` to always install the XDP SDK (at least on Linux) now?